src/Models/ReferenceTokenDataModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace IdentityServerConfig.Models;
+
+public class ReferenceTokenDataModel
+{
+    public string Status { get; set; } = "notChecked";
```

Comment:
Maybe an enum would be appropriate to represent the status here.

Reply:
fixed in a185da2